src/user/components/SavedLocations.js

```diff
@@ -0,0 +1,261 @@
+import React, { useState, Fragment } from "react";
```

Comment:
Add a tooltip or comment next to 'delete' button to say something like "are you sure you want to do this? You can no longer use this station in future"